src/styles/blocks/_main-cards.scss

```diff
@@ -0,0 +1,27 @@
+.main-cards {
+
+  &__list {
+    margin: 0 auto;
+    display: flex;
+    flex-wrap: wrap;
+    padding: 50px 0 72px;
+    max-width: calcCardsContainerWidth(
+      $width-card,
+      $qty-card-per-row,
+      $margin-right-product-item
+    );
+  }
+
+  &__item {
+    @include marginCardsRightBottom($qty-card-per-row);
+  }
+
+  &__card {
+    margin: 0 auto;
+    transition: $duration-animation all;
```

Comment:
better to define what property do  you need  to transition (scale in this case) 
Fix it everywhere